app/models/metric_rollup.rb

```diff
@@ -1,3 +1,4 @@
+# @see Metric
```

Comment:
>`@see` Metric

```ruby
# @see Metric
# This works in a way similar to Metric.
# Since this table contains long term rollup data, there are 12 child tables, one per month.
# The month is based upon the timestamp column.
```